codegenerator/cli/npm/envio/package.json.tmpl

```diff
@@ -31,6 +31,7 @@
     "envio-darwin-arm64": "${version}"
   },
   "dependencies": {
+    "@clickhouse/client": "1.12.1",
```

Comment:
Will this dep be included with all indexers? Maybe it should be conditionally included